src/nodes/display/FilmNode.js

```diff
@@ -0,0 +1,56 @@
+import TempNode from '../core/TempNode.js';
+import { uv } from '../accessors/UVNode.js';
+// import { luminance } from './ColorAdjustmentNode.js';
+import { addNodeElement, tslFn, nodeObject, /*vec3,*/ vec4 } from '../shadernode/ShaderNode.js';
+import { uniform } from '../core/UniformNode.js';
+import { mix, fract, clamp, rand } from '../math/MathNode.js';
+import { timerLocal } from '../utils/TimerNode.js';
+
+class FilmNode extends TempNode {
+
+	constructor( textureNode, intensity = 0.5, grayscale = false ) {
+
+		super();
+
+		this.textureNode = textureNode;
+
+		this.intensity = uniform( intensity );
```

Comment:
I think that intensity could be an `intensityNode` here, and the user could choose between const/uniform or procedural

Reply:
I've updated the code. Even with `intensityNode` as a property, we default to a uniform node, right?